### PR TITLE
Configure QtCreator: Adds a sentence about gdbserver usage

### PR DESCRIPTION
**Pull request by FlorentRevest**
This clarifies the remote debugging capabilities of Qt Creator.

### CODE REVIEW

File: docs/articles/sdk/configure-qtcreator.rst [hunk 1]
```diff
@@ -157,6 +157,7 @@ How to open the template service project with QtCreator
 * Here you should select the kit to use to build project
 * Select the kit you have already created in the previous step and click ``Configure``
 * You can now compile and deploy to your target using the green arrow on the left of your screen
-
+* You can also remotely debug your program using the green arrow with a gray insect on the left of your screen.
```
Comment (sashko) [inline docs/articles/sdk/configure-qtcreator.rst:160]: grey

Comment (FlorentRevest) [reply docs/articles/sdk/configure-qtcreator.rst:160]: Actually, I checked and relied on http://www.dictionary.com/e/gray-or-grey/
Gray is more american and grey is more british... Not sure which one we should use for consistency, I don't think it has ever been defined

Comment (sashko) [reply docs/articles/sdk/configure-qtcreator.rst:160]: As a fellow European, I am on your side, but we used American English everywhere here :(

**Review by sashko — COMMENTED** [review]
btw, title in present simple, please :-P

File: docs/articles/sdk/configure-qtcreator.rst (hunk 1 above)
Comment (FlorentRevest) [reply docs/articles/sdk/configure-qtcreator.rst:160]: Then, we should stick with gray with an a ... ?

**Pull request merged**